src/overlays/actors/ovl_En_Daiku/z_en_daiku.c

```diff
@@ -197,17 +197,18 @@ void func_809438F8(EnDaiku* this, PlayState* play) {
     }
 
     if (this->unk_278 == ENDAIKU_PARAMS_FF_3) {
-        f32 sq, abs;
+        f32 xzDist;
+        f32 abs;
```

Comment:
```suggestion
        f32 yaw;
```
Note I actually fix this in #1205 (https://github.com/zeldaret/mm/pull/1205#discussion_r1133139110), but it is fine here. I like `xzDist` over `dist` anyways.

Reply:
I knew I had seen something similar before, wasn't sure if it was already merged and this was just a similar instance or not